packages/react/src/elements/components/Toast/Toast.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ToastPresenter from "./ToastPresenter";
+import ToastAnimator from "./ToastAnimator";
+
+function Toast({ in: inProp, ...toastProps }) {
+  return (
+    <ToastAnimator in={inProp}>
+      <ToastPresenter {...toastProps} />
+    </ToastAnimator>
```

Comment:
I pass `ToastPresenter` in here, but it could just as well be `FutureWierdToast` or `ClientAppToastVariant`.